src/edges/edges.js

```diff
@@ -22,27 +22,29 @@ const getInterests = (interests) => {
 
 const getStroke = (shareValues) => {
   const { exact, minimum, maximum } = shareValues || {};
-  if (exact === undefined) {
+  if (exact === undefined && minimum <= maximum) {
```

Comment:
I'm not sure the logic here is quite right in the (admittedly niche) scenario in which `maximum` > `minimum`, as you'll get no value for `getStroke` but `minimum` returned for `getText`. Which I think will lead to a floating text with no edge. Suggest no line and no text in such instances, or (probably better) a default line and 'interest details unknown'